src/stave.ts

```diff
@@ -317,7 +317,7 @@ export class Stave extends Element {
   setText(
     text: string,
     position: number,
-    options: {
+    options?: {
       shift_x: number;
       shift_y: number;
       justification: number;
```

Comment:
You are right this has to be Partial as well(resolved). I did not remove the `ts-nocheck` from stave_tests.ts as there were a lot of changes. 
In general @0xfe always asked to declare inline when the structure was not too complex.

Reply:
I included the changes to resolve stave_test.ts sorry. It would be more work now to remove it because I rebased.
@ronyeh please look into them. There is a visual difference on the bars because the double ones are now correctly displayed.